docs/reference/argocd.md

```diff
@@ -999,6 +1000,30 @@ spec:
   statusBadgeEnabled: true
 ```
 
+## Single sign-on Options
+
+The following properties are available for configuring the Single sign-on component.
+
```

Comment:
please add a note that currently only openshift is supported for keycloak provider.